Pipfile

```diff
@@ -8,7 +8,7 @@ bandit = "==1.7.0"
 black = "==20.8b1"
 isort = "==5.8.0"
 mypy = "==0.812"
-panther-core = "==0.0.9"
+panther-core = "==0.0.10"
```

Comment:
is this right? shouldn't this be 0.1.0?

Reply:
currently the detections engine/executor are using 0.0.10.  Is that going to change in this upcoming release?

Reply:
from release-1.40's requirements.txt
```
panther-core==0.0.10
```

Reply:
ahh I missed that we actually went up to 0.0.10 in the pip .. my bad ... yeah we have some newer versions coming that should probably go out in a separate PR